main/2021/WG-07-14.md

```diff
@@ -46,4 +46,139 @@ None.
 
 ## Meeting Notes
 
-To be added after the meeting.
+### Attendees
+Luke Wagner
+
+Derek Schuff
+
+Deepti Gandluri
+
+Eric Prud’Hommeaux
+
+Conrad Watt
+
+Ryan Hunt
+
+Sergey Rubanov
+
+### Proposals and discussions
+
+#### SIMD to Phase 5
+
+DG: we’ve been testing implementations and tools, no issues have come up with implementations or users, so we thought it would be a good time to poll for phase 5. Should we do a consensus, or more detailed poll?
+
+LW: I expect we can get consensus. Ryan, do yo have any comments from the Firefox side?
+
+RH: I’m not actually a WG member but i’m here to answer questions about FF if anyone has them. We don’t have issues with our implementation, and we’re supportive of moving to phase 5.
+
+DG: any objections then?
+
+<no objections noted>
+
+Consensus. Feature moves to phase 5
+
+#### Publish First Public Working Draft, and auto-publishing
+EP: For our purposes here, “TR” means w3.org TR; it means a doc that conforms to publication rules.
+The previous doc (the rec) is big and unwieldy. I decided to try to write a sphinx to TR tool to convert it automatically: https://github.com/w3c/sphinx-to-tr example output is at http://uu3.org/checkouts/w3c/sphinx-to-tr/out/
+
+It takes a “fake” spec and runs it through ReSpec, which creates a valid TR doc. ReSpec has some material that shows editors, etc. It has a config that has to be edited from time to time. It generates the front matter and table of contents (richer that what sphinx generates)
+
+LW: this looks very nice. The TOC looks actually useful.
+
+EP: the TOC is more detailed, let you refer to numbered sections, etc. It’s scriptable, we can run it as a GH action, and can be integrated into W3C’s auto-publish. So if we get our action set up to output the sphinx, we can hook it up.
+To get this running we have to decide to move to first public working draft (starting over after our REC, basically), and then to decide to do automatic publication.
+
+LW: how does that relate to using an evergreen standard?
+
+EP: technically they are orthogonal. Auto-publication just lets you keep it up to date when you iterate on a working draft. Evergreen is that you can stay in CR, and not have to go to REC every couple years, etc. 
+
+LW: we don’t merge stuff too much, it’s mostly like we just did, with a “really done” chunk of work.
+
+EP: yeah when you go to CR you stabilize for 3 months, to update accessibility, internationalization, keep a summary of what’s changed, etc. With evergreen, the patent commitment and review process you did to get to CR, you want to make sure you don’t invalidate the previous patent/accessibility/i18n reviews etc by starting everything over or whatever.
```

Comment:
s/to update accessibility, internationalization/to give accessibility, internationalization etc. a chance to review/